distribution/docs/src/main/jdocs/content/_developingComponents/developing-sources-contents.adoc

```diff
@@ -0,0 +1,152 @@
+:title: Developing Sources
+:type: developingComponent
+:status: published
+:link: _developing_sources
+:summary: Creating a custom source
+
```

Comment:
These files are unchanged from this point on from the ones in `src/main/resources/_contents`